platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/migration/ImportMigrationContextImpl.java

```diff
@@ -68,55 +67,54 @@
 
   private final Set<String> files = new HashSet<>();
 
-  private final ZipFile zip;
+  private final Map<ZipEntry, byte[]> zipContents;
 
   private final List<InputStream> inputStreams = new ArrayList<>();
 
   /**
    * Creates a new migration context for an import operation representing a system context.
    *
    * @param report the migration report where warnings and errors can be recorded
-   * @param zip the zip file associated with the import
+   * @param zipContents the content from the zip file associated with the import
    * @throws IllegalArgumentException if <code>report</code> or <code>zip</code> is <code>null
    * </code>
    * @throws java.io.IOError if unable to determine ${ddf.home}
    */
-  public ImportMigrationContextImpl(MigrationReport report, ZipFile zip) {
+  public ImportMigrationContextImpl(MigrationReport report, Map<ZipEntry, byte[]> zipContents) {
     super(report);
-    Validate.notNull(zip, ImportMigrationContextImpl.INVALID_NULL_ZIP);
-    this.zip = zip;
+    this.zipContents = zipContents;
   }
 
   /**
    * Creates a new migration context for an import operation.
    *
    * @param report the migration report where warnings and errors can be recorded
-   * @param zip the zip file associated with the import
+   * @param zipContents the content from the zip file associated with the import
    * @param id the migratable id
    * @throws IllegalArgumentException if <code>report</code>, <code>zip</code>, or <code>id</code>
    *     is <code>null</code>
    * @throws java.io.IOError if unable to determine ${ddf.home}
    */
-  public ImportMigrationContextImpl(MigrationReport report, ZipFile zip, String id) {
+  public ImportMigrationContextImpl(
+      MigrationReport report, Map<ZipEntry, byte[]> zipContents, String id) {
     super(report, id);
-    Validate.notNull(zip, ImportMigrationContextImpl.INVALID_NULL_ZIP);
-    this.zip = zip;
+    this.zipContents = zipContents;
   }
 
   /**
    * Creates a new migration context for an import operation.
    *
    * @param report the migration report where warnings and errors can be recorded
-   * @param zip the zip file associated with the import
+   * @param zipContents the contents of the zip file associated with the import
    * @param migratable the migratable this context is for
    * @throws IllegalArgumentException if <code>report</code>, <code>zip</code> or <code>migratable
```

Comment:
comment is out of date ... no more `zip`

Reply:
fixed